docs/UserGuide.md

```diff
@@ -289,6 +293,9 @@ If your changes to the data file makes its format invalid, BookFace will discard
 **Q**: How do I clear the sample data on BookFace?<br>
 **A**: Simply use the `clear all` command.
 
+**Q**: Is it normal for the order of the books to change upon restarting the application?<br>
+**A**: Yes; it is a perfectly normal behaviour for the application **if there are loaned books in the book list**, as the intention is to show books that have been loaned as a priority in the list upon starting the app.
```

Comment:
We can explain why we don't refresh the current UI after `loan`, as because the librarian is already used to the current UI, we don't want to disrupt it by making the book disappear from his view. But then probably a counterargument to this is that `list books` should refresh the UI anyways

Reply:
That's a good explanation! Okay, I will add this into the UserGuide in the next commit.

Reply:
Alright sounds good